doc/configuration.md

```diff
@@ -43,6 +43,8 @@ ENABLE\_NODE\_MONOREPO\_BUILD| Apply node monorepo build if repo indicates it
 COMPRESS\_DESTINATION\_DIR   | Indicates if the entire output directory needs to be compressed.   | ""      | `false` | `true`, `false`
 PRUNE\_DEV\_DEPENDENCIES     | Only the prod dependencies are copied to the output for Node apps. | ""      | `false` | `true`, `false`
 NPM\_REGISTRY\_URL           | Specify the npm registry url.                                | ""      | "http://foobar.com/"
+CONFIG\_YARN\_TIMEOUT           | Specify the yarn timeout config with a delay in milliseconds.                                | ""      | "600000"
```

Comment:
It looks like we're reading `YARN_TIMEOUT_CONFIG` in the code, but the docs here say otherwise -- just want to make sure we're both are pointing to the same environment variable.

Reply:
Updated